classes/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.book = self
```

Comment:
- [ ] the Rentals as my colleague said here is the hint `The add_rental method takes one argument which is the date for when the book was rented out as well as a Book object with information about what book it was and who rented it out on that day.
 The code attempts to initialize a new person object with an age of 18, and a name of "Bob".
 The add_rental method will create a new rental object for the book "War and Peace" on January 1st, 2019.
`